kahuna/public/js/crop/view.html

```diff
@@ -39,7 +39,7 @@
         </div>
 
         <div class="top-bar-item">
-            <button class="button"
+            <button class="button crop__start-crop"
```

Comment:
pedantic point: `crop__state-crop` doesn't reflect all states of the button - the text of the button becomes `cropping` when a crop is in process.